### ISSUE TITLE
Pipfile / requirements.txt

### ISSUE THREAD

**Issue by itdaniher** [christabor/plantstuff#3]
Would be quite useful to have an authoritative list of the required packages - I see pytesseract, PIL, amongst others are imported, but I don't see a list anywhere.

Cheers!

**Comment by christabor** [comment]
Yeah, this "package" is nowhere near anything "production" (whatever that means in this context), so I haven't done much cleanup ✅ but certainly this would be good to have!
